README.md

```diff
@@ -41,7 +41,7 @@ Read ["Installation"] from [The Book].
 
 ## Installing from Source
 
-If you really want to install from source (though this is not recommended), see
```

Comment:
this is intended, as installing from source is not encouraged and not what the average user is supposed to do